workspaces/simon-game/src/app/components/Box.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+type Props = { color?: string };
+
+export function Box(props: Props) {
+  const style = {
+    backgroundColor: props.color || "lightblue",
```

Comment:
Take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing, and let's use it here!

Reply:
Updated with this operator!